### PR TITLE
Seamoth claw arm

### PR DESCRIPTION
**Pull request by josephcrowell**
You should skip the project file. Assemblies are stored in a slightly different place on my machine. Initial selection of claw module still won't toggle until you exit and enter the Seamoth if saved in the Seamoth.

### CODE REVIEW

File: MoreSeamothUpgrades/MonoBehaviours/BetterDrillable.cs [hunk 1]
```diff
@@ -402,7 +402,7 @@ public ItemsContainer GetStorageContainer(Vehicle veh, Pickupable pickupable)
             {
                 var seamoth = (SeaMoth)veh;
 
-                for(int i = 0; i < 4; i++)
+                for(int i = 0; i < 9; i++)
```
Comment (PrimeSonic) [inline MoreSeamothUpgrades/MonoBehaviours/BetterDrillable.cs]: Revert this: 
1. Storage slots are not allowed in the extra slots provided by the Slot Extender [link](https://github.com/Senna7608/SennaSubnauticaPlugins/blob/master/SlotExtender/Patchers/Equipment_Patcher.cs#L25)
2. Do not use a hard-coded number of maximum slots when trying to account for the Slot Extender as there is no guarantee that all players will have that mod installed

File: MoreSeamothUpgrades/MoreSeamothUpgrades.csproj [hunk 2]
```diff
@@ -50,17 +50,19 @@
     <Reference Include="System.Data" />
     <Reference Include="System.Xml" />
     <Reference Include="UnityEngine">
-      <HintPath>D:\Steam Library\steamapps\common\Subnautica\Subnautica_Data\Managed\UnityEngine.dll</HintPath>
+      <HintPath>D:\Steam\steamapps\common\Subnautica\Subnautica_Data\Managed\UnityEngine.dll</HintPath>
```
Comment (PrimeSonic) [inline MoreSeamothUpgrades/MoreSeamothUpgrades.csproj]: As a courtesy to the author, revert changes to DLL reference paths in `csproj` files before submitting the Pull Request.

File: MoreSeamothUpgrades/Patches/Seamoth_Patches.cs [hunk 3]
```diff
@@ -17,14 +17,28 @@ static void Postfix(SeaMoth __instance, int slotID, bool active)
...
+                // If its not null
+                if (seamothClawModule != null)
+                {
+                    // Set its toggle!
+                    seamothClawModule.toggle = active;
+                }
+            }
+
             // If its the SeamothDrillModule
             if (techType == SeamothModule.SeamothDrillModule)
```
Comment (PrimeSonic) [inline MoreSeamothUpgrades/Patches/Seamoth_Patches.cs]: Chain with `else if`

**Review by PrimeSonic — CHANGES_REQUESTED** [review]
Some simple changes you can do right now to ensure a clean merge.

**Comment by ahk1221** [comment]
What PrimeSonic suggested.

File: MoreSeamothUpgrades/MonoBehaviours/BetterDrillable.cs (hunk 1 above)
Comment (josephcrowell) [reply MoreSeamothUpgrades/MonoBehaviours/BetterDrillable.cs]: Ah I see.

**Comment by josephcrowell** [comment]
Requested changes pushed.

**Comment by PrimeSonic** [comment]
Looks good from my end.
@ahk1221 The rest is up to you.

**Pull request merged**